docs/development/core/development-functional-tests.asciidoc

```diff
@@ -8,17 +8,59 @@ We use functional tests to make sure the Kibana UI works as expected. It replace
 
 The `FunctionalTestRunner` is very bare bones and gets most of its functionality from its config file, located at {blob}test/functional/config.js[test/functional/config.js]. If you’re writing a plugin you will have your own config file. See <<development-plugin-functional-tests>> for more info.
 
-Execute the `FunctionalTestRunner`'s script with node.js to run the tests with Kibana's default configuration:
+
+The following will start Kibana, Elasticsearch and the chromedriver for you. To run the functional UI tests use the following commands
```

Comment:
I agree, but that is not correct @ycombinator. I think we should preface these instructions as two ways to run the tests:

 1. Easiest way to run the tests once, much slower when running the tests multiple times:
    - `npm run test:ui`
        - does everything in a single command
        - tears down everything after the tests run
        - exit code reports success/failure of the tests
 1. Optimize test execution for faster development:
    - two commands, run in separate terminals, separates the components that are long-running and slow from those that are ephemeral and fast.
    - `npm run test:ui:server`
      - starts elasticsearch and the kibana server
      - slow to start
      - can be reused for multiple executions of the tests
      - automatically restarts the kibana server when relevant changes are detected
    - `node scripts/functional_test_runner`
      - runs the tests against kibana/elasticsearch that were started `npm run test:ui:server`
      - exit code reports success/failure of the tests